examples/booking_app/lib/feature_hotel/search/blocs/hotel_list_bloc.dart

```diff
@@ -1,8 +1,11 @@
 import 'dart:async';
 
+import 'package:booking_app/feature_hotel/search/models/capacity_filter_data.dart';
+import 'package:booking_app/feature_hotel/search/models/date_range_filter_data.dart';
 import 'package:favorites_advanced_base/core.dart';
 import 'package:favorites_advanced_base/models.dart';
 import 'package:flutter/material.dart';
```

Comment:
@DDavidPrime do we need to import the material package into the business layer?

Reply:
`DateTimeRange` is part of flutter's `material` library, so it is required in this case.